src/java/org/apache/cassandra/net/async/NettyFactory.java

```diff
@@ -113,6 +118,7 @@
                                         "MessagingService-NettyAcceptor-Threads", false);
         inboundGroup = getEventLoopGroup(useEpoll, FBUtilities.getAvailableProcessors(), "MessagingService-NettyInbound-Threads", false);
         outboundGroup = getEventLoopGroup(useEpoll, FBUtilities.getAvailableProcessors(), "MessagingService-NettyOutbound-Threads", true);
+        streamingGroup = getEventLoopGroup(useEpoll, FBUtilities.getAvailableProcessors(), "Streaming-Netty-Threads", false);
```

Comment:
Alternatively, maybe just use one larger group?

I'm not sure what the right answer is for IO ratio. I think that when all the Netty threads do is prepare messages you don't really need to mess with it. We don't actually have non-IO tasks they are all really IO tasks even non-IO tasks since they are serialization/deserialization.